lavamoat/browserify/beta/policy.json

```diff
@@ -1422,6 +1459,9 @@
       }
     },
     "@metamask/keyring-controller": {
+      "globals": {
+        "console.log": true
```

Comment:
For future reviewers: Now required because of:
- https://github.com/MetaMask/core/blob/v305.0.0/packages/keyring-controller/src/KeyringController.ts#L2487